src/js/rx/glossary.js

```diff
@@ -1,5 +1,5 @@
-const glossary = [
-  {
+module.exports = {
+  glossary: [{
```

Comment:
You might consider making this an object instead of an array.  If you keyed it by the `term`, it would eliminate the need for `getGlossaryTerm`.

(It's also possible you want to use the `glossary` in other ways, in which case, ignore me)